CMakeLists.txt

```diff
@@ -58,6 +58,9 @@ option(RDK_INSTALL_DEV_COMPONENT "install libraries and headers" ON)
 option(RDK_USE_BOOST_REGEX "use boost::regex instead of std::regex (needed for systems with g++-4.8)" OFF)
 option(RDK_USE_BOOST_IOSTREAMS "use boost::iostreams" ON)
 option(RDK_BUILD_MINIMAL_LIB "build the minimal RDKit wrapper (for the JS bindings)" OFF)
+
+set(RDK_BOOST_VERSION "1.58.0" CACHE STRING "minimum required version of Boost")
```

Comment:
I wonder if there's any reason not to just make this an `option`?

Reply:
No. I changed it to a normal variable.